deploy/packaging/rpm/centos/6/SOURCES/geowave-init.sh

```diff
@@ -49,7 +49,10 @@ wait_for_start() {
 start() {
   echo -n "Starting $DISPLAY_NAME: "
   if ! is_running; then
-  	runuser -l geowave -c "$GEOWAVE_HOME/geoserver/bin/startup.sh &" > /dev/null 2>&1
+    if [ -f /etc/geowave/geowave.config ]; then
+      SOURCE_CMD="source /etc/geowave/geowave.config;"
```

Comment:
@chrisbennight The config file would not be managed by RPM and if absent you'd be using the default settings. I'd probably use  Puppet to mange the file with something like the following.

```
# Bump up the memory allowed for GeoServer
  file { '/etc/geowave/geowave.config':
    ensure  => file,
    owner   => 'geowave',
    group   => 'geowave',
    mode    => 644,
    content => 'export JAVA_OPTS="-Xmx2500m -XX:MaxPermSize=128m"',
    require => Package['geowave-core'],
    notify  => Service['geowave'],
  }
```